pipelines/rbsa/rbsa.py

```diff
@@ -6,7 +6,8 @@
     undiscount_gas, 
     index_heatcool, 
     normalize_totals,
-    zipcode_correlation
+    zipcode_correlation, 
+    discount_gas
```

Comment:
Make sure when these are GTG and you start merging into master, mitigate the merge conflicts and the ordering of tasks.

Reply:
I'll make sure to pull master and fix after every step